prerequisites/build-functions/edit_GCC_download_prereqs_file_if_necessary.sh

```diff
@@ -14,7 +14,7 @@ replace_wget()
     sed -i${backup_extension} -e '2 a\'$'\n'". ${OPENCOARRAYS_SRC_DIR}/prerequisites/build-functions/ftp_url.sh"$'\n' "${download_prereqs_file}"
```

Comment:
> which is probably rare so I wouldn't worry about this.

Yes, but at some point I'd like to add more QA for our scripting, including automated shellcheck tests. So if there is a way to trigger this code path, then I can simplify/fix/improve this and then test it. Presumably this only gets triggered when building gcc from source. Do you know which OSes will cause the ftp method to be invoked? I guess I might be able to move curl, wget and gcc out of my path and try it on Mac OS? I agree this is a low priority...